server/test/Endpoints/router-test.js

```diff
@@ -1,6 +1,29 @@
-const tape = require('tape');
+const test = require('tape');
+const request = require('supertest');
+const app = require('../../app');
+const dbBuild = require('../../database/config/db_build');
 
-tape('just for testing', (t) => {
-  t.equal(1, 1, 'on is equal one');
-  t.end();
+test('Get student information from /api/v1/student/user/12345', (t) => {
+  dbBuild()
+    .then(() => {
+      request(app)
+        .get('/api/v1/student/user/12345')
+        .expect(200)
+        .expect('Content-Type', /json/)
+        .end((err, res) => {
+          const obj = JSON.parse(res.text);
+          if (err) {
+            t.error(err);
+          } else if (obj.error) {
+            t.error(obj.error);
```

Comment:
No  No No you should check the error if it as you sent it from backend. 
```suggestion
            t.equal(err.error, 'Sorry we have some issues', 'get same data that expected');
```